cupy/core/carray.pxi

```diff
@@ -18,7 +18,7 @@ cdef class CArray(CPointer):
 
     def __init__(self, ndarray arr):
         cdef Py_ssize_t i
-        cdef int ndim = arr.ndim
+        cdef int ndim = arr._shape.size()
```

Comment:
Just curious, what is the benefit by rewriting this way? The two things look the same.